packages/perspective/package.json

```diff
@@ -1,90 +1,91 @@
 {
-    "name": "@jpmorganchase/perspective",
-    "version": "0.2.11",
-    "description": "Perspective.js",
-    "main": "build/perspective.node.js",
-    "browser": "src/js/perspective.parallel.js",
-    "publishConfig": {
-        "access": "public"
-    },
-    "files": [
-        "build/*",
-        "index.d.ts",
-        "src/**/*",
-        "webpack-plugin.js",
-        "babel.config.js"
+  "name": "@jpmorganchase/perspective",
```

Comment:
Only line 84 is new, a websocket heartbeat library, others are just indented changes

Reply:
@LukeSheard I modified the git commit hook to return error code 1 on lint failure, rather than running `yarn fix` - the latter had caused too many issues with missing files.  Unfortunately this currently only checks the JS code - as e.g. [`clang-format` does't easily let you validate](https://stackoverflow.com/questions/22866609/can-clang-format-tell-me-if-formatting-changes-are-necessary)

Reply:
This is outdated now anyway, it looks fine in the PR.